domains/client/domain-operator/src/bundle_processor.rs

```diff
@@ -117,8 +250,7 @@ where
             ))
         })?;
 
-        if local_receipt.invalid_bundles != receipt.invalid_bundles {
-            // TODO: Generate fraud proof
+        if !self.check_receipt_validity(&local_receipt, receipt) {
```

Comment:
According to a previous discussion https://github.com/subspace/subspace/pull/1957#discussion_r1326123748 with @vedhavyas, a bundle with an invalid ER should not be considered as an invalid bundle thus let's remove the check here.

Reply:
Done.